common/entrypoint_mautic_web.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# wait untill the db is fully up before proceeding
+while [[ $(mysqladmin --host=db --user=$MYSQL_USER --password=$MYSQL_PASSWORD ping) != "mysqld is alive" ]]; do
+	sleep 1
```

Comment:
could have here `echo "Waiting Mysql to be ready..." inside the loop. That would help in quite common error case where BD credentials are wrong or missing.

Reply:
@ilkkao 

if the db can't be reached for whatever reason, the error is shown.

wrong host:
```
mysqladmin: connect to server at 'db' failed
error: 'Can't connect to server on 'db' (115)'
Check that mariadbd is running and that the socket: '/run/mysqld/mysqld.sock' exists!
```

wrong credentials:
```
mysqladmin: connect to server at 'db' failed
error: 'Access denied for user 'mautic_db_users'@'172.22.0.3' (using password: YES)'
mysqladmin: connect to server at 'db' failed
error: 'Access denied for user 'mautic_db_users'@'172.22.0.3' (using password: YES)'
```

so I think that's fine?

Reply:
Great, I wrongly assumed there's no output

Reply:
I also fixed the check, so it also takes into account incorrect host and port, see https://github.com/mautic/docker-mautic/pull/238/commits/8a58100de4eb83a09b87623cb8a1c0b7897bbb90